core/domain/platform_parameter_domain.py

```diff
@@ -717,6 +718,7 @@ class PlatformParameter:
         DataTypes.BOOL.value: lambda x: isinstance(x, bool),
         DataTypes.STRING.value: lambda x: isinstance(x, str),
         DataTypes.NUMBER.value: lambda x: isinstance(x, (float, int)),
+        DataTypes.LIST_STR.value: lambda x: isinstance(x, list),
```

Comment:
This is wrong, how does this reject e.g. a list of numbers?

Reply:
Will update that.

Reply:
I have removed the addition of a new type for platform-parameter as we are planning to go ahead with the alternatives for converting some of the config properties to platform parameters.
Thanks